common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -344,10 +344,10 @@ main() {
             ;;
         "status")
             status_function
+            return $?
```

Comment:
What is reason behind this `return`?
`status_function` has unconditional `exit $error` at the end. We won't make it to this line.
Also, why not check return code directly?

Reply:
Oh, right that's how I "solved" it last time, I forgot about that. I prefer to directly exit from the function itself, to make sure the exit code is correct, so I'll simply drop this commit.
Thanks :+1: